_posts/2020-09-14-release-0-21-4.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: post
+title:  "JuMP 0.21.4 is released"
+date:   2020-09-14
+categories: [releases]
+permalink: /blog/0.21.4-release/
+---
+
+We are happy to announce the joint release of JuMP 0.21.4 and MathOptInterface
+0.9.15! These releases are a mix of new features and some much needed
+performance optimizations. This should be a non-breaking release, please let us
+know if this isn't the case by [opening a Github issue](https://github.com/jump-dev/JuMP.jl/issues/new/choose)
+or joining the [Gitter chat](https://gitter.im/JuliaOpt/JuMP-dev).
+
+On the JuMP side, new features of interest include
+* the `relax_integrality` function, which relaxes all integrality constraints in
+the model. This is useful for solving the LP relaxation of a MIP.
+* the `compute_conflict!` function, which exposes the ability of some solvers to
+compute an irreducible infeasible subsystem of an infeasible model. This is very
+useful when debugging, and we expect more work in the future to refine a
+user-friendly API.
+
+We also spent time improving the [time](https://github.com/jump-dev/JuMP.jl/pull/2279)
+[it](https://github.com/jump-dev/JuMP.jl/pull/2284)
+[takes](https://github.com/jump-dev/JuMP.jl/pull/2285)
+[to](https://github.com/jump-dev/JuMP.jl/pull/2286)
+[run](https://github.com/jump-dev/JuMP.jl/pull/2303)
+[the](https://github.com/jump-dev/JuMP.jl/pull/2306)
+[tests](https://github.com/jump-dev/JuMP.jl/pull/2307),
+[fixing bugs](https://github.com/jump-dev/JuMP.jl/pull/2271), and
+[improving error messages](https://github.com/jump-dev/JuMP.jl/pull/2276).
+
+On the MathOptInterface side, we spent the majority of the last few months
+working on performance improvements. These changes include: [faster reading and
+writing of MathOptFormat files](https://github.com/jump-dev/MathOptInterface.jl/pull/1111);
+specialized `copy_to` methods for [Clp](https://github.com/jump-dev/Clp.jl/pull/94)
+and [GLPK](https://github.com/jump-dev/GLPK.jl/pull/143), which reduces the time
+taken to copy the problem from JuMP into the solver; and numerous minor changes
+to the backend of MathOptInterface, such as specialized dictionaries which act
+like arrays at first, [but switch to dictionaries when elements are deleted](https://github.com/jump-dev/MathOptInterface.jl/pull/1142).
+
+[@blegat](https://github.com/blegat) also implemented an option that changes the
+way to compute which bridges to use in a transformation, opening the way for us
+to experiment with different bridging schemes in the future.
+
+On the solver front, the conic solver SCS in particular received a
+[lot](https://github.com/jump-dev/SCS.jl/pull/182)
```

Comment:
A reader shouldn't have to click through all these links to see what the improvements are. It would be more useful to have some summary, like model load time decreased by a factor of X on one of our benchmarks.